pkg/backup/handlers/arango/backup/arango_client_impl.go

```diff
@@ -201,6 +202,8 @@ func (ac *arangoClientBackupImpl) Progress(jobID driver.BackupTransferJobID) (Ar
 			completedCount++
 		case driver.TransferAcknowledged:
 		case driver.TransferStarted:
+		case "":
+			completedCount++
```

Comment:
Be aware that empty string is the initial state. The operator might ask for the state and concludes that the upload has completed before anything was done.

Reply:
Add additional check if total is more than 0